### ISSUE TITLE
Address bar should always open

### ISSUE THREAD

**Issue by raj123** [minbrowser/min#998]
In some cases, I need address bar should always open.

**Comment by PalmerAL** [comment]
Can you explain more what you mean? What situation would you need this, and what would it look like?

(can you draw a picture?)

**Comment by raj123** [comment]
situation is: when I am taking screenshot using any tool, in chrome i capture address bar along with screen.

As the address bar is not visible by default, the url will not be visible in screenshot. I think I am clear.

**Comment by PalmerAL** [comment]
We have a `!screenshot` command; maybe we could put the URL in the filename if you use that?

**Comment by raj123** [comment]
I tried !screenshot in address bar and it is doing nothing..other bangs shortcut are working properly. 
Thanks

**Comment by PalmerAL** [comment]
You're right, it seems to have stopped working at some point. Fixed in f8022aae7fa93db86819bb61a961cd4a1891114e.

It looks like https://github.com/electron/electron/issues/18709 would need to be fixed to make the filename change though.

**Comment by raj123** [comment]
yes this is working now.. Thanks
